sagemaker_neo_compilation_jobs/gluoncv_yolo_darknet/train_yolo.py

```diff
@@ -549,9 +466,7 @@ def transform_fn(net, data, content_type, output_content_type):
     x, image = gcv.data.transforms.presets.yolo.transform_test(mx.nd.array(data), 320)
     
     #check if GPUs area available
```

Comment:
same thing here, still valid?

Reply:
Thanks, changed the comment in the following commit.